FlowCrypt/Functionality/Services/Organisational Rules Service/OrganisationalRulesService.swift

```diff
@@ -63,4 +65,17 @@ extension OrganisationalRulesService: OrganisationalRulesServiceType {
             resolve(organisationalRules)
         }
     }
+
+    func getSavedOrganisationalRulesForCurrentUser() -> OrganisationalRules? {
```

Comment:
I would probably make this a required ` -> OrganisationalRules {` . If it's pulled before it's there, than that's a code error, and should crash the app.

Reply:
done